cudapoa/src/allocate_block.hpp

```diff
@@ -251,6 +263,8 @@ class BatchBlock
         offset_d_ += cudautils::align<int64_t, 8>(sizeof(*graph_details_d->sorted_poa) * max_nodes_per_window_ * max_poas_);
         graph_details_d->sorted_poa_node_map = reinterpret_cast<decltype(graph_details_d->sorted_poa_node_map)>(&block_data_d_[offset_d_]);
         offset_d_ += cudautils::align<int64_t, 8>(sizeof(*graph_details_d->sorted_poa_node_map) * max_nodes_per_window_ * max_poas_);
+        graph_details_d->node_distance_to_head = reinterpret_cast<decltype(graph_details_d->node_distance_to_head)>(&block_data_d_[offset_d_]);
```

Comment:
Should this be fenced with adaptive flag?

Reply:
yes, this was added before the other arrays (band_start ...), that's why it's left separate, but no reason to keep it that way. fixed.
As a next step, another change in `allocate_block` is to consider banded-alignment storage for adaptive-banding and add a memory check in adaptive NW kernel to ensure that assumption is not violated, otherwise throw an error code.